From linux-kernel-owner+greg=40kroah.com-S1750928AbVJ2Kbw@vger.kernel.org Sat Oct 29 03:35:46 2005
Date: Sat, 29 Oct 2005 13:31:49 +0300 (EEST)
From: Meelis Roos <mroos@linux.ee>
Subject: PCI: Fix VIA 686 PCI quirk names
Message-ID: <Pine.SOC.4.61.0510291327500.23616@math.ut.ee>

The quirk names for VIA 686 are mistyped in 2.6.14 (686 vs 868). S3 868 
influence? :) Here is a patch to correct them.

Signed-off-by: Meelis Roos <mroos@linux.ee>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/pci/quirks.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/pci/quirks.c	2005-11-02 09:25:02.000000000 -0800
+++ gregkh-2.6/drivers/pci/quirks.c	2005-11-02 11:53:52.000000000 -0800
@@ -462,11 +462,11 @@
 
 	pci_read_config_word(dev, 0x70, &hm);
 	hm &= PCI_BASE_ADDRESS_IO_MASK;
-	quirk_io_region(dev, hm, 128, PCI_BRIDGE_RESOURCES + 1, "vt82c868 HW-mon");
+	quirk_io_region(dev, hm, 128, PCI_BRIDGE_RESOURCES + 1, "vt82c686 HW-mon");
 
 	pci_read_config_dword(dev, 0x90, &smb);
 	smb &= PCI_BASE_ADDRESS_IO_MASK;
-	quirk_io_region(dev, smb, 16, PCI_BRIDGE_RESOURCES + 2, "vt82c868 SMB");
+	quirk_io_region(dev, smb, 16, PCI_BRIDGE_RESOURCES + 2, "vt82c686 SMB");
 }
 DECLARE_PCI_FIXUP_HEADER(PCI_VENDOR_ID_VIA,	PCI_DEVICE_ID_VIA_82C686_4,	quirk_vt82c686_acpi );
 
